samples/openmp_testsuite/Makefile

```diff
@@ -0,0 +1,46 @@
+TOP=$(abspath ../..)
+include $(TOP)/defs.mak
+
+CONTAINER_NAME=omp.myst.apline.062021
+CONTAINER_IMAGE_FLAVOR=alpine
+CONTAINER_IMAGE_VERSION=3.8
+GCC_VERSION=7.5.0
+
+ID=$(shell docker run -d $(CONTAINER_NAME))
+
+all: appdir rootfs
+
+EXT2FS_SIZE=2147483648
+
+rootfs:
+	sudo $(MYST) mkext2 --force --size $(EXT2FS_SIZE) appdir rootfs
+	sudo truncate --size=-4096 rootfs
+
+appdir:
+	docker build -t $(CONTAINER_NAME) . \
+	--build-arg IMAGE_FLAVOR=$(CONTAINER_IMAGE_FLAVOR) \
+	--build-arg IMAGE_VERSION=$(CONTAINER_IMAGE_VERSION) \
+	--build-arg GCC_VERSION=$(GCC_VERSION)
+	$(MAKE) export
+
+export:
```

Comment:
We have a tool `appbuilder` does exactly the same things.

Reply:
Changed custom build logic to use  `appbuilder` instead.